src/controllers/claims/get.controller.ts

```diff
@@ -1,6 +1,11 @@
-import {Request, Response} from "express";
+import { Request, Response } from 'express';
+import WalletService from '@/services/WalletService';
 
 export const FindWallet = async (_req: Request, res: Response) => {
-    };
-
-
+    const response = await WalletService.isWalletExisting(_req.params.wallet);
+    if (response) {
+        res.status(200).send(true);
+    } else {
+        res.status(200).send(false);
+    }
```

Comment:
This can be condensed to one line.

res.status(200).send(response !== undefined);

Reply:
Response isn't undefined when there are no records returned... After logging the response, it appears WalletService returns null.